index.html

```diff
@@ -103,19 +103,22 @@
 
       <!-- Radio buttons for colour palettes -->
       <div>
+        <form id="palette-form"></form>
         <h4>Palette</h4>
         <label>
-          <input type="radio" name="vision" value="normal" checked /> Normal
+          <input type="radio" name="vision" value="normal" checked /> Norm
```

Comment:
I don't think it is a good idea to shorten the lengths of these names as it makes what type they are unclear

Reply:
Reinstated